OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,57 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+struct dynamically_sized_perf_event_callchain_sample {
+  struct dynamically_sized_perf_event_callchain {
+    uint64_t nr;
+    std::unique_ptr<uint64_t[]> ips;
```

Comment:
Again, I don't really agree with this being a different type and name than the one in the manpage. The point of the other "dynamically_sized" struct is to have something very similar to the actual record but without having to allocate 64 KB. If you need something different like here, you can just have fields directly in the `PerfEvent` class, similarly to how it's done for `PerfEventSampleRaw`. 
What you are doing here is indeed similar to what is done for `PerfEventSampleRaw` and is in fact the opposite of `dynamically_sized_perf_event_sample_stack_user`, where we have a fixed sized record but we want to allocate only what is necessary.

Reply:
Okay, I see what you mean. I changed it such that it is more similar to PerfEventSampleRaw.
Hope its better now.